docs/source/notes/distributed_autograd.rst

```diff
@@ -295,6 +298,84 @@ you can refer to **Distributed Autograd Algorithm Smart mode** section in the
 
 Distributed Optimizer
 ^^^^^^^^^^^^^^^^^^^^^
-Coming soon...
+
+The :class:`~torch.distributed.optim.DistributedOptimizer` operates as follows:
+
+1. Takes a list of remote parameters (:class:`~torch.distributed.rpc.RRef`) to
+   optimize. These could also be local parameters wrapped within a local
+   ``RRef``.
+2. Takes a :class:`~torch.optim.Optimizer` class as the local
+   optimizer to run on all nodes.
+3. The distributed optimizer creates an instance of the local ``Optimizer`` on
```

Comment:
is 3 kind of duplicated with 2?

Reply:
2 is mentioning the inputs to the distributed optimizer, 3 is just mentioning that we use those to create RRefs remotely.